cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255, required=True)
+    description = serializers.CharField(required=False)

```

Comment:
The `description` field should use `serializers.CharField` with `allow_blank=True` instead of `required=False` to better reflect the model's `blank=True` option.

Reply:
Done.